terraform/modules/github_actions_access/openid_connect.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_iam_openid_connect_provider" "main" {
+  url = "https://token.actions.githubusercontent.com"
+
+  client_id_list = ["sts.amazonaws.com"]
+
+  thumbprint_list = ["6938fd4d98bab03faadb97b34396831e3780aea1",
```

Comment:
are these static?

Reply:
Good question. I stole these from Delta which has this comment:
> Mixed information about the requirement for a thumbprint and its value but based on this post: https://github.blog/changelog/2023-06-27-github-actions-update-on-oidc-integration-with-aws/ have added the values below.

After some digging it looks like these are being ignored, the [ReadMe](https://github.com/aws-actions/configure-aws-credentials) for the relevant Github Action says: 
>Note that the thumbprint below has been set to all F's because the thumbprint is not used when authenticating token.actions.githubusercontent.com. This is a special case used only when GitHub's OIDC is authenticating to IAM. IAM uses its library of trusted CAs to authenticate. The value is still the API, so it must be specified.

So I'll update to the same w/ a comment pointing at that ReadMe